aten/src/ATen/native/quantized/cpu/qlinear_dynamic.cpp

```diff
@@ -404,9 +404,6 @@ static auto registry =
             torch::RegisterOperators::options()
                 .kernel<QLinearDynamicInt8<true>>(DispatchKey::CPUTensorId))
         .op("quantized::linear_dynamic_fp16(Tensor X, Tensor W_prepack) -> Tensor Y",
-            torch::RegisterOperators::options()
-                .kernel<QLinearDynamicFp16<true>>(DispatchKey::CPUTensorId))
-        .op("quantized::linear_relu_dynamic_fp16(Tensor X, Tensor W_prepack) -> Tensor Y",
             torch::RegisterOperators::options()
                 .kernel<QLinearDynamicFp16<true>>(DispatchKey::CPUTensorId));
```

Comment:
If `template <bool ReluFused>` in Line 339 is removed, here we don't need to specify `<true>`.